src/main/java/com/gregtechceu/gtceu/client/ClientProxy.java

```diff
@@ -57,6 +60,9 @@ public static void init() {
             ClientCacheManager.registerClientCache(GTClientCache.instance, "gtceu");
             Layers.registerLayer(OreRenderLayer::new, "ore_veins");
             Layers.registerLayer(FluidRenderLayer::new, "bedrock_fluids");
+            if (LDLib.isModLoaded(GTValues.MODID_FTB_CHUNKS)) {
+                FTBChunksPlugin.getInstance().initialize();
```

Comment:
I'm not sure this is the right place to initialize? When I tried to run this branch, I got a NPE at `dev.ftb.mods.ftbchunks.api.FTBChunksAPI.clientApi(FTBChunksAPI.java:42) ~[ftb-chunks-forge-314906-5956390.jar%23210!/:2001.3.4]`. I'm not sure if my mods loaded in a different order from you or something? I was just using `gradle :runClient`.

Using an event listener instead fixed this for me. (This is the event [FTB-XMod-Compat](https://github.com/FTBTeam/FTB-XMod-Compat/blob/main/common/src/main/java/dev/ftb/mods/ftbxmodcompat/FTBXModCompat.java#L39) listens for, although they use architectury for events.)
```java
@SubscribeEvent
public void ftbChunksSetup(FMLCommonSetupEvent event) {
    if (LDLib.isModLoaded(GTValues.MODID_FTB_CHUNKS)) {
        FTBChunksPlugin.getInstance().initialize();
    }
}
```

Reply:
yeah, super weird, when I build and add the jar to an instance it works fine, but calling `runClient` gets you a NPE.
maybe it is a load order issue with the dev environment.
I'll move it to the event listener.

Reply:
Actually, my event listener version results in the options getting reset every time the game is restarted - maybe that part needs to load earlier, or maybe the options should live in GTM's config instead of adding them to FTB chunks.

Reply:
it's not your event listener, I was baffled by that too, ftb registers all their properties in the static block of the client config interface which is loaded when they call init from the client api class.
our properties are saved properly, but when they call init they override any property that wasn't registered on that config object by that point (which are just their properties from the static block), to solve it I tried mixin into the static block of the config interface, but that's not possible with the current state of mixins.
the solution was just doing it before the init call in the client api class.